modules/apps/remote-app/remote-app-web/src/main/resources/META-INF/resources/admin/edit_remote_app_entry.jsp

```diff
@@ -81,32 +81,41 @@ renderResponse.setTitle(editRemoteAppEntryDisplayContext.getTitle());
 					<aui:validator name="customElementName" />
 				</aui:input>
 
-				<%
-				for (String customElementURL : editRemoteAppEntryDisplayContext.getCustomElementURLs()) {
-				%>
+				<div id="<%= liferayPortletResponse.getNamespace() + "_type_customElementURLs" %>">
 
-					<div class="repeatable">
-						<aui:input ignoreRequestValue="<%= true %>" label="url" name="customElementURLs" type="text" value="<%= customElementURL %>">
-							<%--<aui:validator name="url" />--%>
-						</aui:input>
-					</div>
+					<%
+					for (String customElementURL : editRemoteAppEntryDisplayContext.getCustomElementURLs()) {
+					%>
 
-				<%
-				}
+						<div class="lfr-form-row">
+							<aui:input ignoreRequestValue="<%= true %>" label="url" name="customElementURLs" type="text" value="<%= customElementURL %>">
+								<%--<aui:validator name="url" />--%>
+							</aui:input>
+						</div>
 
-				for (String customElementCSSURL : editRemoteAppEntryDisplayContext.getCustomElementCSSURLs()) {
-				%>
+					<%
+					}
+					%>
 
-					<div class="repeatable">
-						<aui:input ignoreRequestValue="<%= true %>" label="css-url" name="customElementCSSURLs" type="text" value="<%= customElementCSSURL %>">
-							<%--<aui:validator name="url" />--%>
-						</aui:input>
-					</div>
+				</div>
 
-				<%
-				}
-				%>
+				<div id="<%= liferayPortletResponse.getNamespace() + "_type_customElementCSSURLs" %>">
 
+					<%
+					for (String customElementCSSURL : editRemoteAppEntryDisplayContext.getCustomElementCSSURLs()) {
+					%>
+
+						<div class="lfr-form-row">
+							<aui:input ignoreRequestValue="<%= true %>" label="css-url" name="customElementCSSURLs" type="text" value="<%= customElementCSSURL %>">
+								<%--<aui:validator name="url" />--%>
```

Comment:
just reminding to remove this comment here 🎗️

Reply:
Brian commented it personally so I think it's fine, it's fixed in the next PR :D